release-notes.md

```diff
@@ -24,6 +38,17 @@ The following updates were made in this release:
 
 * `requestEventIdentifier` is now appended to all non-track events so that Lifecycle (or other extension events that are sent to Analytics) can be viewed with rich detail in Project Griffon.
 
+## January 24, 2020
+
+The following updates were made in this release:
+
+**Android Griffon 1.1.1**
+
+* Griffon SDK now reports event's source, type, sequence number and timestamp for every event.
+* Griffon SDK adds and reports uniqueIdentifier associated with eventHub events. (works from Core v1.4.2)
```

Comment:
Please reword as follows:
"* The Griffon SDK adds and reports the `uniqueIdentifier` that is associated with eventHub events. 
     This update is effective from Mobile Core version 1.4.2.